src/tests/e2e/pages/login.page.js

```diff
@@ -29,71 +29,95 @@ const sleep = milliseconds => {
 class LoginPage {
   async login (username, password, mockLogin = false) {
     log.debug(username, password)
-    // assume we are at login page
-
+    
+    let currentPageCount = (await browser.pages()).length
```

Comment:
move inside if(!mocklogin){...}, this is only relevant if mock user is not used

Reply:
fixed